base_local_planner/include/base_local_planner/goal_functions.h

```diff
@@ -53,6 +53,15 @@
 
 namespace base_local_planner {
 
+  /**
+   * Compute the direction of a point from the normal passing through the goal
+   *
+   * @param global_pose The pose of the robot in the global frame
+   * @param global_plan The plan being followed
```

Comment:
S:
```suggestion
   * @param pose The pose of the robot in the global frame
   * @param path The path being followed
```

Reply:
I just followed the comment style of the other functions in the codebase. Most of them call the arguments global pose and global path

Reply:
:ok: